msal-dotnet-articles/acquiring-tokens/desktop-mobile/wam.md

```diff
@@ -7,44 +7,69 @@ description: "MSAL is able to call Web Account Manager (WAM), a Windows 10+ comp
 
 MSAL is able to call Web Account Manager (WAM), a Windows 10+ component that ships with the OS. This component acts as an authentication broker allowing the users of your app benefit from integration with accounts known to Windows, such as the account you signed into your Windows session.
 
-## New WAM Preview in MSAL 4.44+
```

Comment:
I would delete this page and keep only https://learn.microsoft.com/en-us/azure/active-directory/develop/scenario-desktop-acquire-token-wam (for which I have a PR to update it)

Reply:
@bgavrilMS let's take this into a new pull request.